### PR TITLE
Swift proxy read loadbalancer IP correctly from kubernetes service

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

**Review by katrybacka — APPROVED** [review]

**Comment by kaweue** [comment]
recheck ha

**Comment by kaweue** [comment]
recheck ha

File: pkg/controller/swiftproxy/swiftproxy_controller.go [hunk 1]
```diff
@@ -143,7 +143,7 @@ func (r *ReconcileSwiftProxy) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, nil
 	}
 	swiftProxy.Status.ClusterIP = svc.Spec.ClusterIP
-	swiftProxy.Status.LoadBalancerIP = svc.Spec.LoadBalancerIP
+	swiftProxy.Status.LoadBalancerIP = getLoadBalancerIP(svc)
```
Comment (shifali19) [inline pkg/controller/swiftproxy/swiftproxy_controller.go:146]: May be we could create this Swift proxy service using our k8s/service package.

**Review by shifali19 — CHANGES_REQUESTED** [review]

File: pkg/controller/swiftproxy/swiftproxy_controller.go (hunk 1 above)
Comment (kaweue) [reply pkg/controller/swiftproxy/swiftproxy_controller.go:146]: hmm that is a good idea :)

**Review by shifali19 — APPROVED** [review]

**Comment by kaweue** [comment]
recheck

**Comment by kaweue** [comment]
recheck

**Review by katrybacka — APPROVED** [review]

**Comment by kaweue** [comment]
I also tested it on CaVA and loadbalancer IP of proxy service is registered in keystone

**Pull request merged**